snuba/web/delete_query.py

```diff
@@ -27,8 +28,25 @@ def construct_condition(body: Dict[str, Any]) -> Expression:
     return combine_and_conditions(and_conditions)
 
 
-def delete_query(
-    storage: WritableTableStorage, table: str, body: Dict[str, Any]
+def delete_from_storage(
+    storage: WritableTableStorage, conditions: Dict[str, Any]
```

Comment:
I think we will want to pass through `tenant_ids` and other information aside from the `conditions` in the future because we want logging around where the delete queries are coming from, but for now this works